packages/http-client-java/CONTRIBUTING.md

```diff
@@ -46,6 +46,20 @@ Internally, it does
 1. Build the `@typespec/http-client-java` tgz locally. See [Build and Pack](#build-and-pack).
 2. Re-install the tgz (along with other dependencies) in the test module.
 
+### Debug `.ts` code
+
+1. In vscode, toggle `Auto Attach` as `Always` (click `ctrl+shift+p` and input `Toggle Auto Attach`, choose the option `Always`). You may need to restart vscode after setting.
+2. Run below commands
+
+```
+$ cd packages/http-client-java
+$ npm link
+$ cd packages/http-client-java/generator/http-client-generator-test
+$ npm link ../../ # this will link packages/http-client-java/http-client-generator-test to packages/http-client-java, so that you can debug emitter's .ts code directly
```

Comment:
Adding this part because I found debugging `.ts` file is much more convenient than debugging `.js` file during development. And this is the easiest way I found to debug it.
I would like to write it down in case later others need to debug `.ts` file or I forgot how to debug `.ts` file.

Let me know if you don't want me to include this in our `CONTRIBUTING.md`. I can also remove it and write down in my local.